index.php

```diff
@@ -22,6 +24,7 @@
         <div>
```

Comment:
I ran the HTML through the validator and it picked up some errors. Would recommend running through HTML validator and check with Charlie if they are strict errors which need sorting (as PHP code is being prioritised) .